tests/testthat/test-checks.R

```diff
@@ -179,6 +179,23 @@ test_that("workflow objects", {
   })
 })
 
+test_that("errors informatively with needed package install", {
+  # rstanarm is not installed during CI runs
+  # in contexts where it _is_ installed, skip the test.
```

Comment:
It took me a second to realize what happened here. A rewording would be nice. Maybe:

`errors informatively with needed package isn't installed`